File: LeetCode/1791. Find Center of Star Graph/1791.cs
public class Solution {
    public int FindCenter(int[][] edges) {

        // The center of the star graph is the node that is connected to all other nodes.
        // So, we can just check the first two edges and return the common node.
        return edges[0][0] == edges[1][0] || edges[0][0] == edges[1][1] ? edges[0][0] : edges[0][1];
        
        
    }
}

File: LeetCode/1791. Find Center of Star Graph/1791.java
class Solution {
    public int findCenter(int[][] edges) {
        // Compare the nodes of the first two edges to find the common one
        if (edges[0][0] == edges[1][0] || edges[0][0] == edges[1][1]) {
            return edges[0][0];
        } else {
            return edges[0][1];
        }
    }
}
